asyn/asynPortDriver/asynPortDriver.cpp

```diff
@@ -56,10 +56,12 @@ class paramList {
     asynStatus setUInt32(int index, epicsUInt32 value, epicsUInt32 valueMask, epicsUInt32 interruptMask);
     asynStatus setDouble(int index, double value);
     asynStatus setString(int index, const char *string);
+    asynStatus setString(int index, std::string& string);
```

Comment:
setter type methods should take a const reference when the argument isn't modified (the same way the other overload takes 'const char*' instead of 'char*').

```
asynStatus setString(int index, const std::string& string);
```